rust/rsc/src/rsc/add_job.rs

```diff
@@ -29,8 +29,8 @@ pub async fn add_job(
         stdin: Set(payload.stdin),
         is_atty: Set(payload.is_atty),
         hidden_info: Set(payload.hidden_info),
-        stdout: Set(payload.stdout),
-        stderr: Set(payload.stderr),
+        stdout_id: Set(payload.stdout_id),
+        stderr_id: Set(payload.stderr_id),
```

Comment:
minor nit: Both here and above in the DB, I think "stdfoo_blob_id" or "stdfoo_blob_key" or something like that would be better. Just seeing "stdout_id" doesn't really tell me whats happening here.